src/core/xfa/bind.js

```diff
@@ -635,6 +635,13 @@ class Binder {
             /* skipConsumed = */ this.emptyMerge
           ).next().value;
           if (!match) {
+            // If there is no match (no data) and occur.min === 0
+            // container is entirely excluded.
+            // https://www.pdfa.org/norm-refs/XFA-3_3.pdf
```

Comment:
That document is over 1500 pages long, please link directly to the relevant section instead; I'm guessing that https://www.pdfa.org/norm-refs/XFA-3_3.pdf#G23.1904923 might be correct here?

Reply:
I think this section is a bit more specific:
https://www.pdfa.org/norm-refs/XFA-3_3.pdf#G12.1428332